src/lbpDashboard/lbpDashboard.html

```diff
@@ -26,4 +27,4 @@
         <token-distribution start-date.bind='lbpMgr.startDate' token-details.bind='lbpMgr.metadata.tokenDetails'></token-distribution>
       </div>
     </div>
-</template>
+</template>
```

Comment:
would be nice if you could adjust you vscode settings to not remove the line feed at the end. @wantedsystem